pkg/gtest/gocovertest.go

```diff
@@ -54,6 +56,13 @@ func (g *GocoverTest) EnsureGoTestFiles() error {
 	for _, c := range changes {
 		// in case of other files, except go files
 		if !strings.HasSuffix(c.FileName, ".go") {
+			fmt.Fprintf(g.Writer, "skip checking for other file: %s\n", c.FileName)
+			continue
+		}
+
+		// don't generate test file for go mock files
+		if goMockFileRegexp.MatchString(c.FileName) {
```

Comment:
will this ignore the mock_ directory too?

Reply:
ask this because some folder is like
mock_aaa/a.go

Reply:
or 
aa/mock_a.go

Reply:
File path that contains `mock_` will all be ignored.